objects/Tank.java

```diff
@@ -57,10 +31,10 @@ public void move(Commands.Direction cmd)
         switch (cmd)
         {
           case UP:
-            rowPos--;
+            if(! isCollision(mainRenderer, getRowPos() - 1, getColPos())) setRowPos(getRowPos() - 1);
```

Comment:
I don't prefer this kind of if statement. Wouldn't it be better if the condition and statement is split into separate lines?

Reply:
I have made the if condition into multiple lines